patches/minecraft/net/minecraft/util/datafix/codec/DatapackCodec.java.patch

```diff
@@ -0,0 +1,12 @@
+--- a/net/minecraft/util/datafix/codec/DatapackCodec.java
++++ b/net/minecraft/util/datafix/codec/DatapackCodec.java
+@@ -29,4 +29,9 @@
+    public List<String> func_234887_b_() {
+       return this.field_234883_d_;
+    }
++
++   @Deprecated //Forge internal
++   public static DatapackCodec getModdedDefault() {
```

Comment:
Again, why are you adding a random method onto this random class instead of say... anywhere else we stick random functions like ForgeHooks?